docs/issue_labeling.md

```diff
@@ -0,0 +1,60 @@
+# Issue and Feature Labeling
```

Comment:
should we add it under this dev section (https://github.com/lyft/amundsen/blob/master/mkdocs.yml#L61) of our doc?

Reply:
That makes sense, I guess I just need to add another entry on that yml file?

Reply:
correct, then follow https://github.com/lyft/amundsen/blob/master/deploy_website.sh#L8-L12 to test locally on the doc side and verify.

Reply:
Awesome, thanks!

Reply:
I like the labeling.

How will it work for non-Lyft contributors? Currently I think it’s only available for you people. Maybe it’s just a GitHub setting adjustment or something else needed ...

Reply:
Oh, true, thanks for the heads up @jornh!

We'll probably start with keeping it only for Lyft developers, but it might have sense to open it up.